.github/workflows/build.yml

```diff
@@ -66,8 +67,10 @@ jobs:
             docker.io/unfoldedcircle/r2-pyinstaller:${PYTHON_VER} \
             bash -c \
               "cd /workspace && \
-              python -m pip install -r requirements.txt && \
-              pyinstaller --clean --onedir --name intg-denonavr --add-data intg-denonavr/locales:locales intg-denonavr/driver.py"
+              pip install --user -r requirements.txt && \
+              pip install --user ./denonavrlib && \
+              pip install --user --force-reinstall pyinstaller && \
+              PYTHONPATH=~/.local/lib/python3.11/site-packages:\$PYTHONPATH ~/.local/bin/pyinstaller --clean --onedir --name intg-denonavr --add-data intg-denonavr/locales:locales intg-denonavr/driver.py"
```

Comment:
Is this change required?

If yes, can we avoid somehow to use the hard-coded `~/.local/lib/python3.11` path?
It's only a matter of time until the `unfoldedcircle/r2-pyinstaller` image is updated to Python 3.12 or newer, then this will fail.

Does `--user` work with the [pre-cached library installation in the build image](https://github.com/unfoldedcircle/ucr2-pyinstaller/blob/0bff3b0bff8140256bad17bd3bf2cc6d639f452d/Dockerfile#L14), or does it re-install & compile all libraries again in the user install directory?
This saves a lot of build time on x64 machines, especially on slow GitHub runners. It shouldn't matter where the libraries are installed in the throw-away image.

Note to myself: build image needs to be updated with the new library versions :-)